### PR TITLE
log rpm-ostree status, and fix hack/

### PR DESCRIPTION
**Pull request by cgwalters**

### CODE REVIEW

**Comment by cgwalters** [comment]
Example output:

```
$ oc logs pods/machine-config-daemon-22cbx 
I0222 15:16:21.552853  100538 start.go:52] Version: 3.11.0-715-gfbc03999-dirty
I0222 15:16:21.553033  100538 start.go:88] starting node writer
I0222 15:16:21.562475  100538 run.go:22] Running captured: chroot /rootfs rpm-ostree status --json
I0222 15:16:21.690581  100538 daemon.go:168] Booted osImageURL: registry.svc.ci.openshift.org/openshift/origin-v4.0-2019-02-21-043513@sha256:660061d6eae3ee6d93ca836cd52e6033f1d611c629c1ce47cf272c9e9bda2488 (47.318)
I0222 15:16:21.690771  100538 daemon.go:240] Managing node: osiris-worker-0-tpf2x
I0222 15:16:21.706625  100538 start.go:146] Calling chroot("/rootfs")
I0222 15:16:21.706679  100538 run.go:22] Running captured: rpm-ostree status
I0222 15:16:21.752663  100538 start.go:161] State: idle
AutomaticUpdates: disabled
Deployments:
* pivot://registry.svc.ci.openshift.org/openshift/origin-v4.0-2019-02-21-043513@sha256:660061d6eae3ee6d93ca836cd52e6033f1d611c629c1ce47cf272c9e9bda2488
              CustomOrigin: Managed by pivot tool
                   Version: 47.318 (2019-02-16T04:05:17Z)

  pivot://registry.svc.ci.openshift.org/rhcos/maipo@sha256:e1fa79427fbd98704d8429704b45e3b9f6d402fcf4ce9849ecbb74793a70f9db
              CustomOrigin: Provisioned from oscontainer
                   Version: 47.326 (2019-02-21T04:46:33Z)
I0222 15:16:21.765987  100538 daemon.go:494] Current+desired config: worker-e4bf08262ee82a7f0d28943f59be0040
I0222 15:16:21.769832  100538 daemon.go:615] Validated on-disk state
I0222 15:16:21.769862  100538 daemon.go:647] In desired config worker-e4bf08262ee82a7f0d28943f59be0040
I0222 15:16:21.769886  100538 start.go:171] Starting MachineConfigDaemon
I0222 15:16:21.769895  100538 daemon.go:303] Enabling Kubelet Healthz Monitor
```

File: cmd/machine-config-daemon/start.go [hunk 1]
```diff
@@ -154,6 +154,12 @@ func runStartCmd(cmd *cobra.Command, args []string) {
 	}
 
 	if startOpts.onceFrom == "" {
+		status, err := dn.NodeUpdaterClient.GetStatus()
+		if err != nil {
+			glog.Fatalf("unable to get status: %s", err)
```
Comment (jlebon) [inline cmd/machine-config-daemon/start.go]: Minor: "unable to get *rpm-ostree* status" ?

**Comment by ashcrow** [comment]
```
# github.com/openshift/machine-config-operator/pkg/daemon
pkg/daemon/update_test.go:33:20: cannot use testClient (type RpmOstreeClientMock) as type NodeUpdaterClient in field value:
	RpmOstreeClientMock does not implement NodeUpdaterClient (missing GetStatus method)
pkg/daemon/update_test.go:205:20: cannot use testClient (type RpmOstreeClientMock) as type NodeUpdaterClient in field value:
	RpmOstreeClientMock does not implement NodeUpdaterClient (missing GetStatus method)
pkg/daemon/update_test.go:287:20: cannot use testClient (type RpmOstreeClientMock) as type NodeUpdaterClient in field value:
	RpmOstreeClientMock does not implement NodeUpdaterClient (missing GetStatus method)
```

File: cmd/machine-config-daemon/start.go (hunk 1 above)
Comment (cgwalters) [reply cmd/machine-config-daemon/start.go]: Sure, done.

**Comment by ashcrow** [comment]
```
level=error msg="\t* aws_route.to_nat_gw.0: Error creating route: timeout while waiting for state to become 'success' (timeout: 2m0s)
```

**Comment by runcom** [comment]
Kubeapiserver operator failing somehow

/retest

**Pull request merged**